recipes/zeromq/all/patches/0001-cmake-minimum-requirement.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
is this file intentional?

Reply:
Yeah, without the patch, it fails to build on Windows.